### PR TITLE
CU-240jm9f - StateQueryClient - query currentEpoch

### PR DESCRIPTION
**Pull request by AndrewWestberg**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -3,8 +3,8 @@ import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
 plugins {
     java
     id("com.github.ben-manes.versions") version "0.42.0"
-    kotlin("jvm") version "1.6.10"
-    kotlin("plugin.serialization") version "1.6.10"
+    kotlin("jvm") version "1.6.20"
+    kotlin("plugin.serialization") version "1.6.20"
```
Comment (j-mateo) [inline build.gradle.kts:7]: Should we pull the versions from the `Versions` object below?

**Review by j-mateo — APPROVED** [review]
👍🏽  lgtm

File: build.gradle.kts (hunk 1 above)
Comment (AndrewWestberg) [reply build.gradle.kts:7]: https://stackoverflow.com/a/60170218

**Comment by j-mateo** [comment]
> @j-mateo Did you figure out why this PR didn't move the ticket into a different status in clickup?

I missed a step in the integration (adding the repo to the clickup app). Hopefully it's good to go now

**Comment by j-mateo** [comment]
Task linked: [CU-240jm9f StateQueryClient - query currentEpoch](https://app.clickup.com/t/240jm9f)

**Pull request merged**